### PR TITLE
Update instructions for Xdebug 3

### PR DESCRIPTION
**Pull request by mitogh**
### Description of the Change

Added instructions to setup `xdebug 3`

### Checklist:

- [ ] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by dwelch2344** [comment]
This is fantastic. Been trying to figure it out all day, and the instructions worked great for me. Nice to finally debug in VSCode (and share that ability with the rest of our team)

File: README.md [hunk 1]
```diff
@@ -225,6 +320,7 @@ Make sure your IDE is listening for PHP debug connections and set up a path mapp
 2. Install the [PHP Debug](https://marketplace.visualstudio.com/items?itemName=felixfbecker.php-debug) extension.
 3. In your project, go to the Debug view, click "Add Configuration..." and choose PHP environment. A new launch configuration will be created for you.
 4. Set the `pathMappings` parameter to your local `wordpress` directory. Example:
+
 ```json
 "configurations": [
```
Comment (tylercherpak) [inline README.md:325]: @mitogh can we just delete this section since its duplicated above?

Comment (mitogh) [reply README.md:325]: Partially tho, as the main difference between versions 2 and 3 is the port. We can keep either and just mention the fact that the port should be updated appropriately instead.

Comment (tylercherpak) [reply README.md:325]: xdebug2 is no longer supported, its just xdebug3 now right? so I think we can remove it.

Comment (tylercherpak) [reply README.md:325]: actually you are correct, xdebug2 is still used for 7.1 and lower

**Review by tylercherpak — APPROVED** [review]

**Pull request merged**